### ISSUE TITLE
Exception in thread 

### ISSUE THREAD

**Issue by AiratHalitov** [N3TC4T/InstaBrute#5]
```
Exception in thread Thread-21:
Traceback (most recent call last):
  File "C:\Users\airat\AppData\Local\Programs\Python\Python35\lib\threading.py", line 914, in _bootstrap_inner
    self.run()
  File "C:\Users\airat\AppData\Local\Programs\Python\Python35\lib\threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "instabrute.py", line 23, in check_proxy
    proxy = q.get(False)
  File "C:\Users\airat\AppData\Local\Programs\Python\Python35\lib\multiprocessing\queues.py", line 100, in get
    raise Empty
queue.Empty
```

Log 

```
***************************************
* Inastgarm Brute forcer              *
* Coded by N3TC@T                     *
* netcat[dot]av[at]gmail[dot]com      *
***************************************
[+] Username Loaded: usename
[+] Words Loaded: 5
[+] Proxy Loaded: 26

[-] Testing Proxy List...
 --[+]  88.151.182.74:3128  | PASS
 --[+]  80.242.219.50:3128  | PASS
 --[+]  92.47.195.250:3128  | PASS
 --[+]  178.91.246.125:3128  | PASS

[+] Online Proxy:  4
[+] Getting CSRF Token: 
[+] CSRF Token : Nkc7KGlPVyb015JkGkZDv7TvDsyPmgBi 


[!] Initializing Workers
[!] Start Cracking ... 

[!] Error in request
[!] Error in request
[!] Error in request
.....

```

**Comment by N3TC4T** [comment]
fixed in 7271bab20e5bc47ff6c30b193bb65cc4dc85d802

**Comment by AiratHalitov** [comment]
@N3TC4T 
thanks!
